From 599d81835f7c7b130cf38c0b61c5f27b3164283b Mon Sep 17 00:00:00 2001
From: Ankit Agrawal <ankit.agarwal@intel.com>
Date: Thu, 18 May 2023 15:53:48 +0530
Subject: [PATCH] WA-Fixed Build Error in gallium.

Build Error-:
ld.lld: error: version script assignment of 'global' to symbol '__driDriverExtensions' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'nouveau_drm_screen_create' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'radeon_drm_winsys_create' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'amdgpu_winsys_create' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'fd_drm_screen_create' failed: symbol not defined
ld.lld: error: version script assignment of 'global' to symbol 'ac_init_shared_llvm_once' failed: symbol not defined

Above symbols were included in Global flag. but it seems
that it was not defined globally. So removed from global.
By default it will be locally.

Tracked-On: OAM-106853
Signed-off-by: Ankit Agrawal <ankit.agarwal@intel.com>
---
 src/gallium/targets/dri/dri.sym | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/gallium/targets/dri/dri.sym b/src/gallium/targets/dri/dri.sym
index 7f7a93b02ff..13df604d1d9 100644
--- a/src/gallium/targets/dri/dri.sym
+++ b/src/gallium/targets/dri/dri.sym
@@ -1,12 +1,6 @@
 {
 	global:
-		__driDriverExtensions;
 		__driDriverGetExtensions*;
-		nouveau_drm_screen_create;
-		radeon_drm_winsys_create;
-		amdgpu_winsys_create;
-		fd_drm_screen_create;
-		ac_init_shared_llvm_once;
 	local:
 		*;
 };
-- 
2.17.1

